python/servo/testing_commands.py

```diff
@@ -321,12 +321,10 @@ def test_wpt_android(self, build_type: BuildType, binary_args=None, **kwargs):
         )
         return self._test_wpt(build_type=build_type, android=True, **kwargs)
 
-    def _test_wpt(self, build_type: BuildType, with_asan=False, android=False, **kwargs):
-        if not android:
-            os.environ.update(self.build_env())
```

Comment:
Why is it possible to remove this now? This allows passing environment variables into the WPT runner.

Reply:
This is a stray change that should have not have been in the PR. I've restored it now, but all the tests on CI were passing on my fork even without this change. Is the use case here to pass the GStreamer related environment variables on Mac, from `build_env()` to the runner?

Reply:
I think the idea is that if the WPT runner is reading environment variables, they will be passed properly to the runner.

I'm fine with removing it here if it isn't used on CI. If we find a situation where we need to pass environment variables, we can re-add it with a comment (and probably also for Android as well).

Reply:
The environment is not passed as a parameter to `wpt.run.run_tests` and `run_test` directly reads from the global `os.environ`, so the only impact of removing this logic should be that the additional variables added by `build_env()` will no longer be passed to the runner. If the runner was previously reading other variables from parent environment, it should still be able to do that without this logic, so I'll remove it.